IdentityCore/src/cache/serializers/MSIDKeyedArchiverSerializer.m

```diff
@@ -99,4 +125,20 @@ - (MSIDCredentialCacheItem *)deserializeCredentialCacheItem:(NSData *)data
     return nil;
 }
 
+#pragma mark - Class Mapping
+
+- (void)addEncodeClassMapping:(NSDictionary *)classMap
```

Comment:
Will broker have access to keyed archiver serializer instance used by legacy accessor? I think it was private. Maybe we can just set it globally for PRT inside the broker?

Reply:
Since we are going to move `ADBrokerPrimaryRefreshToken` to common core, we don't need to access the serializer of legacy accessor anymore. Instead, we set the mapping of `ADBrokerPRTCacheItem` right in MSIDKeyedArchiverSerializer.m, just like other legacy mapping (`ADUserInformation`...)